controllers/operatorpolicy_controller.go

```diff
@@ -316,7 +318,24 @@ func (r *OperatorPolicyReconciler) buildResources(ctx context.Context, policy *p
 
 	validationErrors := make([]error, 0)
 
-	sub, subErr := buildSubscription(policy, r.DefaultNamespace)
+	disableTemplates := false
+
+	if disableAnnotation, ok := policy.GetAnnotations()["policy.open-cluster-management.io/disable-templates"]; ok {
```

Comment:
I don't like that this is an annotation instead of a spec field but I guess consistency with configuration policy is probably better than changing this at this point.

Reply:
Agreed :laughing: